modules/ipi-install-configuring-ncsi-disable-poweroff.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ipi-install-configuring-ncsi-disable-poweroff_{context}"]
+= Configuring NC-SI and DisablePowerOff for shared NICs
+
+The Network Controller Sideband Interface (NC-SI) enables the Baseboard Management Controller (BMC) to share a system NIC with the host for management traffic, using protocols like Redfish, IPMI, or vendor-specific interfaces. The `DisablePowerOff` feature prevents hard power-offs, ensuring soft reboots to maintain BMC connectivity.
+
+*Prerequisites*
+
+* NC-SI-capable hardware and NICs.
+* BMC configured with an IP address and network connection.
+* Administrative access to the BMC.
+* Access to the OpenShift cluster with `cluster-admin` privileges.
+
+*Procedure*
+
+. Configure the BMC to enable NC-SI for a shared NIC.
+
+. Verify BMC connectivity using Redfish or IPMI by running one of the following commands:
++
+[source,terminal]
+----
+$ curl -k https://<bmc_ip>/redfish/v1/Systems/1
+----
++
+[source,terminal]
+----
+$ ipmitool -I lanplus -H <bmc_ip> -U <user> -P <pass> power status
+----
+
+. Enable the `DisablePowerOff` feature by editing the `Machine` or the `BareMetalHost` resource in the `openshift-machine-api` namespace:
```

Comment:
I'm not 100% sure about the editing the `Machine` part,  let me check with my team about this.

Reply:
@johnwilkins I've double check with Derek, since he did most of the implementation (but now he is in another team), we don't have this available for the `Machine`, so we can remove such information.

Reply:
Thanks @iurygregory. I've removed the metal portion, and placed the module with the BMO docs, since it uses the BMO exclusively now. Let me know if this is good to go.